cppcache/integration/framework/Cluster.h

```diff
@@ -171,60 +171,77 @@ class Cluster {
               const std::string truststorePassword);
 
   bool useSsl();
-  bool requireSslAuthentication();
-  std::string keystore();
-  std::string truststore();
-  std::string keystorePassword();
-  std::string truststorePassword();
 
-  Gfsh &getGfsh();
+  void usePropertiesFile(const std::string propertiesFile);
+  void useSecurityPropertiesFile(const std::string securityPropertiesFile);
+  void useHostNameForClients(const std::string hostNameForClients);
+  bool usePropertiesFile();
+  bool useSecurityPropertiesFile();
+  bool useHostNameForClients();
+bool requireSslAuthentication();
```

Comment:
Something I think happened with your auto formatter, or at least the indentation seems to have gotten out of whack

Reply:
Actually, this test is disabled because it will require a real SNI gateway server. The tricky portion of the test that starts up the cluster and SNI proxy via docker does actually work.

The thought was to get the test checked into develop just to simplify handoff as people go on vacation. Good catch on the formatting. Fixing that now.

Reply:
Good catch. Looks like tabs set incorrectly.